configs/app/ui/views/address.ts

```diff
@@ -58,9 +59,18 @@ const extraVerificationMethods: Array<SmartContractVerificationMethodExtra> = ((
     return SMART_CONTRACT_EXTRA_VERIFICATION_METHODS;
   }
 
-  const parsedMethods = parseEnvJson<Array<SmartContractVerificationMethodExtra>>(getEnvValue('NEXT_PUBLIC_VIEWS_CONTRACT_EXTRA_VERIFICATION_METHODS')) || [];
+  const parsedMethods = parseEnvJson<Array<SmartContractVerificationMethodExtra>>(envValue) || [];
 
-  return SMART_CONTRACT_EXTRA_VERIFICATION_METHODS.filter((method) => parsedMethods.includes(method));
+  return parsedMethods.filter((method) => SMART_CONTRACT_EXTRA_VERIFICATION_METHODS.includes(method));
+})();
+
+const languageFilters: Array<VerifiedContractsFilter> = (() => {
+  const envValue = parseEnvJson<Array<VerifiedContractsFilter>>(getEnvValue('NEXT_PUBLIC_VIEWS_CONTRACT_LANGUAGE_FILTERS'));
+  if (!envValue) {
+    return SMART_CONTRACT_LANGUAGE_FILTERS.filter((filter) => filter !== 'scilla');
```

Comment:
it's not clear why filtering like that, maybe just add another constant SMART_CONTRACT_LANGUAGE_FILTERS_DEFAULT or something?